From: Simon McVittie <smcv@debian.org>
Date: Thu, 8 Jun 2017 22:34:51 +0100
Subject: Remove library dependencies from pkg-config metadata

pkg-config metadata is solely about what is needed to develop against
a particular package, and has no relation to what is needed at runtime;
it is not a substitute for RPM or dpkg metadata.

Signed-off-by: Simon McVittie <smcv@debian.org>
Forwarded: https://github.com/flatpak/xdg-desktop-portal/pull/112
---
 xdg-desktop-portal.pc.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/xdg-desktop-portal.pc.in b/xdg-desktop-portal.pc.in
index ddf0387..2c4d407 100644
--- a/xdg-desktop-portal.pc.in
+++ b/xdg-desktop-portal.pc.in
@@ -10,5 +10,3 @@ interfaces_dir=${datadir}/dbus-1/interfaces/
 Name: xdg-desktop-portal
 Description: Desktop integration portal
 Version: @VERSION@
-Requires: glib-2.0 gio-2.0 flatpak
-Requires.private: gio-unix-2.0
